repos/spack_repo/builtin/packages/pandoc/package.py

```diff
@@ -50,22 +57,43 @@ class Pandoc(Package):
             url="https://github.com/jgm/pandoc/releases/download/2.7.3/pandoc-2.7.3-linux.tar.gz",
         )
 
+    elif platform.system() == "Darwin" and platform.machine() == "aarch64":
+        url = "https://github.com/jgm/pandoc/releases/download/2.14.0.3/pandoc-3.7.0.2-arm64-macOS.zip"
```

Comment:
I assume the version has to be in both locations:

```suggestion
        url = "https://github.com/jgm/pandoc/releases/download/3.7.0.2/pandoc-3.7.0.2-arm64-macOS.zip"
```